### PR TITLE
docs: gitlab-ce-docker docs enhancement

### PR DESCRIPTION
**Pull request by daniel-hutao**
Signed-off-by: Daniel Hu <tao.hu@merico.dev>

## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [ ] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

## Description

gitlab-ce-docker docs enhancement

## Related Issues

N/A

## New Behavior (screenshots if needed)

N/A

### CODE REVIEW

File: docs/plugins/gitlab-ce-docker.md [hunk 1]
```diff
@@ -1,37 +1,47 @@
...
+This plugin installs [GitLab](https://about.gitlab.com/) CE(Community Edition) on Docker.
 
-This plugin installs [Gitlab-CE](https://about.gitlab.com/) in an existing docker, and the container name is `gitlab`.
 ## Usage
 
 Note: 
-1. the user must be `root` or in `docker` group.
-2. https not support now(todo).
+1. the user you are using must be `root` or is in `docker` group;
+2. `https` don't be supported now.
```
Comment (aFlyBird0) [inline docs/plugins/gitlab-ce-docker.md:9]: don't be-> is not ?

File: docs/plugins/gitlab-ce-docker.md [hunk 2]
```diff
@@ -1,37 +1,47 @@
...
+```
+
+1. ssh
+
+```shell
+# port is 22
+git clone git@${YOUR_HOSTNAME}/${YOUR_USERNAME}/${YOUR_PROJECT_NAME}.git
+# port is not 22, 2022 as a sample
+git clone ssh://git@${YOUR_HOSTNAME}:2022/${YOUR_USERNAME}/${YOUR_PROJECT_NAME}.git
+```
```
Comment (aFlyBird0) [inline docs/plugins/gitlab-ce-docker.md]: cool

**Review by aFlyBird0 — APPROVED** [review]

**Review by iyear — APPROVED** [review]

**Comment by aFlyBird0** [comment]
The CI `links when pr` is failed. You could add a rule `.*${.*}.*` to `/.lycheegnore` to fix it.

**Review by aFlyBird0 — APPROVED** [review]

**Comment by daniel-hutao** [comment]
close #948

**Pull request merged**